packages/angular-query-experimental/src/__tests__/inject-queries.test.ts

```
@@ -76,4 +113,548 @@ describe('injectQueries', () => {
+          {
+            queryKey: key2,
+            queryFn: async () => {
+              await new Promise((r) => setTimeout(r, 100))
+              count++
+              return count
+            },
+          },
+        ],
+        combine: (queryResults) => {
+          return {
+            refetch: () => queryResults.forEach((r) => r.refetch()),
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)